ledger-tool/src/output.rs

```diff
@@ -551,6 +561,92 @@ pub fn output_sorted_program_ids(program_ids: HashMap<Pubkey, u64>) {
     }
 }
 
+pub struct AccountsOutputStreamer {
+    account_scanner: AccountsScanner,
+    total_accounts_stats: Rc<RefCell<TotalAccountsStats>>,
+}
+
+impl AccountsOutputStreamer {
+    pub fn new(
+        bank: Arc<Bank>,
+        include_sysvars: bool,
+        include_account_contents: bool,
+        include_account_data: bool,
+        account_data_encoding: UiAccountEncoding,
```

Comment:
these are probably candidates for a config struct

Reply:
Fair enough - done in https://github.com/solana-labs/solana/pull/34915/commits/47fb5850b37e1b21f6a6e27f2fcd5b3e5eaaa430